ddr/include/ddr/std/sstream.hpp

```diff
@@ -25,18 +25,8 @@
 #include "ddr/config.hpp"
 
 #if defined(J9ZOS390)
-
 #include <ctype.h>
-#undef toupper
-#undef tolower
-
-#include <sstream>
-
-#define toupper(c)     (islower(c) ? (c & _XUPPER_ASCII) : c)
-#define tolower(c)     (isupper(c) ? (c | _XLOWER_ASCII) : c)
-
-#else /* defined(J9ZOS390) */
-#include <sstream>
 #endif /* defined(J9ZOS390) */
 
+#include <sstream>
```

Comment:
This was previously not included on z/OS: Have you verified that it is now safe to include?

Reply:
In the before code, I am seeing `#include <sstream>` in the `J9ZOS390` clause already, inbetween the undef and defines.

That being said, it seems to be safe to include so far while compiling as well.

Reply:
Sorry, I misread the change. You are correct, `sstream` was included on all platforms before and should continue to be.